pkg/flavors/flavors.go

```diff
@@ -75,6 +75,6 @@ func CreateConfig(flavor string, params map[string]string) (string, error) {
 		return "", fmt.Errorf("Supplied template for flavor %s is incorrect", flavor)
 	}
 	var buf bytes.Buffer
-	tmpl.Execute(&buf, data) //nolint
+	tmpl.Execute(&buf, flavorData) //nolint
```

Comment:
Why asymmetry in the way errors get handled?

Error from `tmpl.Parse()` is reported.

Error from `tmpl.Execute()` is silenced.

Reply:
Moved this code to postgres flavor and added error checking for both functions.